index.html

```diff
@@ -518,68 +515,72 @@ <h4 data-i18n="index.page.requirements.updates.header">Frequent Updates</h4>
         var elem = document.getElementById(elementID);
         elem.href = _url;
         elem.onclick = function (e) {
-            e.preventDefault();
-            var downloadStarted = false;
-            setTimeout(function () { if (!downloadStarted) window.location.href = _url; }, 1200); // fallback
-            _gaq.push(tagObj, function () {
-                downloadStarted = true;
-                setTimeout(function () { window.location.href = _url; }, 1000); // short delay just in case it doesn't get logged
-            });
+            if (e.which === 2) { // middle button => current tab will stay open => will be logged in any case
```

Comment:
Where did you look? According to https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/which, every MouseEvent has a `.which` property and it's more standardised than `.button`.